.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 app/routes/sessionToken.js
 public/lib
+*.swp
```

Comment:
This is for temp files that vim creates.  You can see that I accidentally commited one in a previous PR :)